addons/crm/views/res_partner_views.xml

```diff
@@ -111,7 +111,7 @@
                             name="schedule_meeting"
                             icon="fa-calendar"
                             groups="sales_team.group_sale_salesman"
-                            context="{'partner_id': active_id, 'partner_name': name}">
+                            context="{'partner_id': active_id, 'partner_name': name, 'default_partner_ids': [active_id], 'search_default_partner_ids': [active_id]}">
```

Comment:
Isn't `default_partner_ids` overriden in:

https://github.com/odoo/odoo/blob/a12309797b3f7140ec8733ed2a6d09ee640c20b1/addons/crm/models/res_partner.py#L56


?

Reply:
indeed, was not needed